pkg/rc/farm.go

```diff
@@ -228,7 +229,7 @@ START_LOOP:
 				go func(id fields.ID) {
 					defer func() {
 						if r := recover(); r != nil {
-							err := util.Errorf("Caught panic in rc farm: %s", r)
+							err := util.Errorf("Caught panic in rc farm: %s\n%s", r, debug.Stack())
```

Comment:
This is interesting: `util.Errorf()` already captures a full stack trace—it just isn't used anywhere. Could there be a more general way to report the stack? Perhaps not, because that method is used everywhere as a generic `error` wrapper, but a full stack trace would only be useful in specific places like this.